kolibri/core/tasks/compat.py

```diff
@@ -1,28 +1,53 @@
+import multiprocessing
+import threading
+
+from concurrent import futures
+
 from kolibri.utils.conf import OPTIONS
 
-try:
-    if not OPTIONS["Tasks"]["USE_WORKER_MULTIPROCESSING"]:
-        raise ImportError()
-    # Import in order to check if multiprocessing is supported on this platform
-    from multiprocessing import synchronize  # noqa
 
-    # Proxy Process to Thread to allow seamless substitution
-    from multiprocessing import Process as Thread  # noqa
-    from multiprocessing import Event  # noqa
+def use_multiprocessing():
+    try:
+        if not OPTIONS["Tasks"]["USE_WORKER_MULTIPROCESSING"]:
+            raise ImportError()
+        # Import in order to check if multiprocessing is supported on this platform
+        from multiprocessing import synchronize  # noqa
+
+        return True
+    except ImportError:
+        return False
```

Comment:
not totally clear to me how to exercise these code paths. Perhaps we should have a gherkin story which ensure both get tested during release?

Reply:
The entire task runner test suite (https://github.com/learningequality/kolibri/tree/b87f9da274a6a9218419cf9f858be695aa04304a/kolibri/core/tasks/test/taskrunner) gets run on both of these configuration options, so I have confidence that these code paths are being exercised regularly.